picard/tagger.py

```diff
@@ -292,7 +292,7 @@ def mb_login(self, callback, parent=None):
         else:
             callback(False)
 
-    def on_mb_authorization_finished(self, callback, successful):
+    def on_mb_authorization_finished(self, callback, successful=False):
```

Comment:
Why the default value here?

Reply:
To make it a kw arg, so it's possible to add other arguments without breaking code calling this callback. Not strictly necessary, but since i changed calling code to use `callback(successful=value)` it makes more sense imho,  where `callback` is a `partial`.
See https://github.com/metabrainz/picard/blob/8608bf16e8f1fc66df1f1dfd64fd4ebe6b75539b/picard/tagger.py#L289-L291